tests/migration/online_migration/yast2_migration.pm

```diff
@@ -209,10 +209,15 @@ sub run {
     assert_screen 'yast2-migration-target-list-selected', 60;
     send_key_until_needlematch 'migration-target-' . get_var("VERSION"), 'down', 20, 3;
     send_key "alt-n";
-    # in leap to sle migration, we need click trust for package-hub's import-untrusted-gpg-key
-    if (is_leap_migration) {
+    # migration via smt will install packagehub and NVIDIA compute, we need click trust
+    # gpg keys; Same with leap to sle migration, need to trust packagehub gpg key.
+    if ((get_var('SMT_URL') =~ /smt/) || (is_leap_migration)) {
         assert_screen 'import-untrusted-gpg-key', 60;
         send_key 'alt-t';
+        if ((check_var('ARCH', 'x86_64')) && (!(is_leap_migration)) || (check_var('ARCH', 'aarch64'))) {
```

Comment:
why we need !(is_leap_migration)  this condition ?

Reply:
Before I change this code, leap_migration has been here. It is also necessary to import packagehub gpg key in migration from leap to sle, because packagehub is in the bless module list.

Reply:
then i suggest using following:  
if ((!is_leap_migration) && ((check_var('ARCH', 'x86_64')) || (check_var('ARCH', 'aarch64')))) {

Reply:
May I know the reason please? if ((check_var('ARCH', 'x86_64')) && (!(is_leap_migration)) means exclude leap migration in x86_64 arch cases.